exercises/Exercises.sln

```diff
@@ -219,6 +219,7 @@ EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "RotationalCipher", "rotational-cipher\RotationalCipher.fsproj", "{91D0A5E0-E39C-472B-87FB-6DD17336AA22}"
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "IsbnVerifier", "isbn-verifier\IsbnVerifier.fsproj", "{531D1AB8-1297-4DE2-9B9B-3742A648EDB7}"
```

Comment:
@ErikSchierboom is there a reason that `IsbnVerifier` no longer has an `EndProject`?

Reply:
Whoops, merge error! I've fixed it.